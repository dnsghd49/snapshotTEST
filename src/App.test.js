import { render, screen } from '@testing-library/react';
import App from './App';
import Renderer from 'react-test-renderer';

test('renders learn react link', () => {
  render(<App />);
  const linkElement = screen.getByText(/learn react/i);
  expect(linkElement).toBeInTheDocument();
});

test('rednering snapshot', () => {
  const testRenderer = Renderer.create(<App />).toJSON
  expect(testRenderer).toMatchSnapshot()
})