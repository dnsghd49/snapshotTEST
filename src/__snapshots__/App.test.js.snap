// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rednering snapshot 1`] = `[Function]`;
